src/main/java/edu/ie3/datamodel/io/source/influxdb/InfluxDbWeatherSource.java

```diff
@@ -172,29 +172,34 @@ public InfluxDbWeatherSource(
 
   private String createQueryStringForIntervalAndCoordinate(
       ClosedInterval<ZonedDateTime> timeInterval, int coordinateId) {
-    return createQueryStringForInterval(timeInterval)
+    return BASIC_QUERY_STRING
+        + " where "
```

Comment:
You might consider introducing a constant for `" where "`. It's not too vital, but reduces the number of code smells...